config/hotspot.config

```diff
@@ -85,6 +90,11 @@ Type: full
 Architectures: aarch64 x86_64 ppc64le s390x
 Directory: 8/jre/debian
 
+Build: releases nightly
+Type: full
```

Comment:
I am not familiar with this file too much. Why we adding `Type: full` instead `Type: full slim` ?

Reply:
ok, got it from https://github.com/AdoptOpenJDK/openjdk-docker/#legend